src/env.d/99sdc-vmadm

```diff
@@ -0,0 +1 @@
+MANPATH=/native/usr/share/man
```

Comment:
This file should be named something like `sdc-vmtools`. `vmadm` is a specific utility that only works in the global zone.

Reply:
Done